app/javascript/chat/components/ChannelFilterButton.jsx

```diff
@@ -0,0 +1,25 @@
+import { h } from 'preact';
+import PropTypes from 'prop-types';
+import { Button } from '@crayons';
+
+export const ChannelFilterButton = ({ type, name, active, onClick }) => {
```

Comment:
Preempting @nickytonline here: all new components have JSDoc, we probably want that here too.